src/status_im/contexts/settings/wallet/events.cljs

```diff
@@ -53,3 +54,86 @@
      handle-connection)))
 
 (rf/reg-event-fx :wallet/get-key-pair-export-connection get-key-pair-export-connection)
+
+(defn extract-keypair-name
+  [db key-uids-set]
+  (when (= (count key-uids-set) 1)
+    (let [key-uid  (first key-uids-set)
+          keypairs (get-in db [:wallet :keypairs])]
+      (->> (filter #(= (:key-uid %) key-uid) keypairs)
+           first
+           :name))))
+
+(defn update-accounts
+  [accounts key-uids-set]
+  (into {}
+        (map (fn [[k account]]
+               (if (and (contains? key-uids-set (:key-uid account))
+                        (= (:operable account) :no))
+                 [k (assoc account :operable :fully)]
+                 [k account]))
+             accounts)))
+
+(defn update-keypairs
+  [keypairs key-uids-set]
+  (map (fn [keypair]
+         (if (contains? key-uids-set (:key-uid keypair))
+           (update keypair
+                   :accounts
+                   (fn [accounts]
+                     (map (fn [account]
+                            (if (and (contains? key-uids-set (:key-uid account))
+                                     (= (:operable account) ":no"))
+                              (assoc account :operable ":fully")
+                              account))
+                          accounts)))
+           keypair))
+       keypairs))
+
+(rf/reg-event-fx :wallet/make-key-pairs-fully-operable
+ (fn [{:keys [db]} [key-uids-to-update]]
+   (let [key-uids-set (set key-uids-to-update)
+         keypair-name (extract-keypair-name db key-uids-set)]
+     {:db (-> db
+              (update-in [:wallet :accounts] update-accounts key-uids-set)
+              (update-in [:wallet :keypairs] update-keypairs key-uids-set))
+      :fx [[:dispatch
+            [:toasts/upsert
+             {:type  :positive
+              :theme :dark
+              :text  (if (= (count key-uids-to-update) 1)
+                       (i18n/label :t/key-pair-imported-successfully {:name keypair-name})
+                       (i18n/label :t/key-pairs-successfully-imported
+                                   {:count (count key-uids-to-update)}))}]]]})))
+
```

Comment:
I think this toast should be displayed in light mode 🌤️

Reply:
Toast colors are intentionally inverted

Reply:
Yup yup that makes sense, but I think we still need to invert the theme for when displaying the toasts in settings.
So maybe we can pass the theme value or compute the inverted theme somewhere?

Reply:
but all settings screen are dark and their toast are light at this moment.

Reply:
Yep the theme should be dark. 👍

Reply:
The Figma designs show light toasts in settings because settings is dark mode-ish (blur theme?) by default. Maybe we can ask the design team to confirm? From what I heard toasts should always be the inverted theme.

And if want to support using this re-frame event from different places in the app where some UI is light mode sometimes (wallet home), then we could try to make this event handler more dynamic when picking the theme.

![image](https://github.com/status-im/status-mobile/assets/2845768/63153505-8b5e-4fbc-afae-6183d9b729bc)

Reply:
Yep yep I get that but if you inspect the toast in designs you'll see that is dark theme. The colors are inverted.
Maybe I'm talking gibberish and if so sorry for this conversation going in circles 😅

Reply:
haha that's okay 🙏, I think I'm confused with how to change the toast to be displayed with the light colors. Atm I think the video attached to this PR shows the dark colors being used in the toast, and I was looking for the way to change that to the light colors.

Reply:
oh maybe we want `:blur? true` here?

Reply:
Nice! thanks for fixing that 🙏